--- extras/check_dependencies.sh	2022-01-15 16:31:04.000000000 +0100
+++ /check_dependencies.sh	2022-01-16 03:22:24.689440339 +0100
@@ -82,22 +82,12 @@
   add_packages libtool
 fi
 
-if ! have svn; then
-  echo "$0: subversion is not installed"
-  add_packages subversion
-fi
-
 if ! have awk; then
   echo "$0: awk is not installed"
   add_packages gawk
 fi
 
 pythonok=true
-if ! have python2.7; then
-  echo "$0: python2.7 is not installed"
-  add_packages python27 python2.7
-  pythonok=false
-fi
 
 if ! have python3; then
   echo "$0: python3 is not installed"
@@ -105,35 +95,6 @@
   pythonok=false
 fi
 
-(
-#Use a subshell so that sourcing env.sh does not have an influence on the rest of the script
-[ -f ./env.sh ] && . ./env.sh
-if $pythonok && ! have python2; then
-  mkdir -p $PWD/python
-  echo "$0: python2.7 is installed, but the python2 binary does not exist." \
-       "Creating a symlink and adding this to tools/env.sh"
-  ln -s $(command -v python2.7) $PWD/python/python2
-  echo "export PATH=$PWD/python:\${PATH}" >> env.sh
-fi
-
-if [[ -f $PWD/python/.use_default_python && -f $PWD/python/python ]]; then
-  rm $PWD/python/python
-fi
-
-if $pythonok && have python && [[ ! -f $PWD/python/.use_default_python ]]; then
-  version=$(python 2>&1 --version | awk '{print $2}')
-  if [[ $version != "2.7"* ]] ; then
-    echo "$0: WARNING python 2.7 is not the default python. We fixed this by" \
-         "adding a correct symlink more prominently on the path."
-    echo " ... If you really want to use python $version as default, add an" \
-         "empty file $PWD/python/.use_default_python and run this script again."
-    mkdir -p $PWD/python
-    ln -s $(command -v python2.7) $PWD/python/python
-    echo "export PATH=$PWD/python:\${PATH}" >> env.sh
-  fi
-fi
-)
-
 mathlib_missing=false
 case $(uname -m) in
   x86_64)  # Suggest MKL on an Intel64 system (not supported on i?86 hosts).
